controllers/profiles/common/properties/managed.go

```diff
@@ -157,28 +158,34 @@ func NewAppPropertyHandler(workflow *operatorapi.SonataFlow, platform *operatora
 			return nil, err
 		}
 		props.Merge(p)
-		p, err = generateKnativeEventingWorkflowProperties(workflow)
+		p, err = resolvePlatformWorkflowProperties(platform)
```

Comment:
So this properties might eventually overwrite the properties configured in the previous lines for the workflow to connect with the DI and JS?

Reply:
I believe that here we are saying that it its not possible https://github.com/apache/incubator-kie-kogito-docs/pull/623/files#diff-f2d457811b79c4bfb0dc80b355343ebf27237be5f64cd51df5fc25b7575d39a7R116

Reply:
Good catch! Let me put it above.